pyteal/ast/method_test.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from pyteal.ast.method import Method
+
+from .. import *
+
+
+def test_method():
+    expr = Method("add(uint64,uint64)uint64")
+    assert expr.type_of() == TealType.bytes
+
+    expected = TealSimpleBlock([TealOp(expr, Op.method, "add(uint64,uint64)uint64")])
```

Comment:
The method name should be surrounded in quotes. Right now it outputs `method add(uint64,uint64)uint64`, but it should be `method "add(uint64,uint64)uint64"`.

See the implementation here: https://github.com/jannotti/go-algorand/blob/168cfe58f1d99b0289accc67098f69482d400295/data/transactions/logic/assembler.go#L655-L671

Reply:
oops...